decentraland/unity-explorer
# fix: lods removing from scene cache

## What does this PR change?

- LODs are added to the scene cache on instantiation. When disposed, they are removed. 
When required again, they were retrieved from the LOD cache, but not added to the scene cache. Generating timeout on the loading screen, since this one never knew that the LODs were ready. This PR adds back to the scene cache when gotten from the LOD cache

- Also, adds an unload immediate on dispose, since profiler is not available at this time and we dont care about hiccups while closing the application.

## How to test the changes?


- Test 1
1. Launch the explorer. Starting in GP
2. Go to 100,100
3. Go to Airdrop Castle (-7,-18)
4. Go to 100,100
5. Go to Airdrop Castle (-7,-718)
6. The second teleports should be almost instantaneous

- Test 2
1. Launch the explorer. Starting in GP
2. Go to ALTIX (113,18)
3. Go back to GP
4. Go to ALTIX (113,18)
5. The second teleports should be almost instantenous

Repeat with any scene you know its a LOD. Do the same process with roads and empty parcels

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
